Curtains/Curtains.Api/Controllers/FiltersCurtainsController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Curtains.Domain.Projections;
+using Curtains.Application.DTO;
+using Curtains.Application.SearchService.Interfaces;
+using Curtains.Infrastructure.SearchQueries;
+
+namespace Curtains.Api.Controllers
+{
+	/// <summary>
+	/// This controller <c> FiltersCurtainsController </c> for indexes 
+    /// and searchind data from ElastiSearch 
+	/// </summary>
+	[Route("api/[controller]")]
+    [ApiController]
+    public class FiltersCurtainsController : ControllerBase
+    {
+        private readonly ICurtainSearchService _searchService;
+
+        public FiltersCurtainsController(ICurtainSearchService searchService)
+        {
+            _searchService = searchService;
+        }
+
+		/// <summary>
```

Comment:
same

Reply:
Looks ok in my IDE

Reply:
In your IDE it can looks like perfect code.
Now we all see that you have troubles with tabulation.
Please check your IDE settings and verify how tabs have been set.
It would be tabs or 4 spaces. It's depends only on your IDE settings.